python/mujoco/__init__.py

```diff
@@ -79,3 +79,8 @@
       from mujoco.egl import GLContext
     else:
       from mujoco.glfw import GLContext
+
+
+from mujoco import mj_versionString
```

Comment:
You don't need this line: the `from mujoco._functions import *` line should've taken care of it already.

Reply:
Done.